README.md

```diff
@@ -1,119 +1,128 @@
-<a name="readme-top"></a>
-<div align="center">
-  <h2><b>The Hello rails app</b></h2>
-</div>
+<!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
-- [📖 \[The Blog app\] ](#-the-blog-app-)
+- [📖 Budget App](#-budget-app)
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
     - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+    - [Video Presentation ](#video-presentation-)
   - [💻 Getting Started ](#-getting-started-)
-    - [Setup](#setup)
     - [Prerequisites](#prerequisites)
-    - [Install](#install)
-  - [👥 Authors ](#-authors-)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
   - [🔭 Future Features ](#-future-features-)
   - [🤝 Contributing ](#-contributing-)
   - [⭐️ Show your support ](#️-show-your-support-)
   - [🙏 Acknowledgments ](#-acknowledgments-)
   - [📝 License ](#-license-)
 
+<!-- PROJECT DESCRIPTION -->
 
+# 📖 Budget App
 
-# 📖 [The Hello rails app] <a name="about-project"></a>
-
-**[The Hello-rails app]** is a template for future RoR projects.
+**The Budget app** is a mobile web application where user can manage the budget.
 
 ## 🛠 Built With <a name="built-with"></a>
-
-- Ruby (IRB)
-- VSCode
-- GitHub
+- ✅ Ruby
+- ✅ Ruby on Rails
+- ✅ Postgresql
+- ✅ Linters
 
 ### Tech Stack <a name="tech-stack"></a>
 
 <details>
-  <summary>Server</summary>
+  <summary>Language</summary>
   <ul>
-    <li><a href="https://guides.rubyonrails.org/getting_started.html">Ruby on Rails</a></li>
+    <li>Ruby</li>
   </ul>
 </details>
 
+<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-- **[  ]**
-- **[  ]**
+- 🔰 **Authenticate users**
+- 🔰 **Splash Welcome page**
+- 🔰 **Read all categories**
+- 🔰 **Add a new category**
+- 🔰 **Read all transactions**
+- 🔰 **Add a new transaction**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
 
-## 💻 Getting Started <a name="getting-started"></a>
+The Live Demo is available [here](https://budget-f9if.onrender.com)
```

Comment:
- [ ] Hey, awesome job deploying your project on Render! 🚀 Unfortunately, it seems there's a glitch at the moment. While the app loads fine, attempting to visit the sign-up page triggers an error. 😟 Could you please run the live version of your project and check the Render logs for any potential errors? 🕵️‍♂️ Let's get that sign-up page issue sorted out. Thanks a bunch! 🙌

![Screenshot from 2024-01-08 20-14-23](https://github.com/zhorabay/Budget/assets/112261237/4a8b5d58-3393-46fb-b3d1-7d1e2c3ccdcb)